examples/experimental/Length of Stay demo on MIMIC/pytorch/trainer.py

```diff
@@ -0,0 +1,102 @@
+"""
+Module containing functions useful for training a pytorch neural network
+"""
+import numpy as np
+import logging
+import torch.optim as optim
+from torch import no_grad
+from torch import manual_seed as torch_manual_seed
+from torch.cuda import manual_seed_all as torch_cuda_manual_seed_all
+import torch.nn.functional as F
+
+
+from pytorch.models import FeedforwardNeuralNetwork
+
+
+LOSSES = {"mean_squared_error": F.mse_loss}
+OPTIMIZERS = {"rmsprop": optim.RMSprop}
```

Comment:
This is good, we might have a lot of them

Reply:
Do you want to add more now or later?

Reply:
Later will be good :)

Reply:
:+1: